setup.py

```diff
@@ -16,8 +16,28 @@
 
 setup(name='haas',
       version='0.2rc2',
+      author='https://github.com/CCI-MOC/haas/network/members',
+      author_email = 'haas-dev-list@bu.edu',
       url='https://github.com/CCI-MOC/haas',
+      description='Hardware-as-a-Service or HaaS is a bare-metal isolation service that automates \
```

Comment:
I would take the name off of this so that it's just `'Bare-metal isolation ... compute resources'`. Also, if you're going to spread this across multiple lines, close the string first. You can do:

```python
'foo bar' \
'baz quux'
```
For example. Much like in C, python will concatinate adjacent string literals.

You probably don't even need the backslash actually, since you're inside a set of parens (provided by the call to `setup`). Give it a try without.